client/signup/steps/woocommerce-install/confirm/index.tsx

```diff
@@ -1,69 +1,33 @@
 import { NextButton } from '@automattic/onboarding';
 import { createInterpolateElement } from '@wordpress/element';
 import { useI18n } from '@wordpress/react-i18n';
-import { ReactElement, useEffect } from 'react';
-import { useSelector, useDispatch } from 'react-redux';
+import { ReactElement } from 'react';
 import { default as HoldList } from 'calypso/blocks/eligibility-warnings/hold-list';
 import WarningList from 'calypso/blocks/eligibility-warnings/warning-list';
 import { LoadingEllipsis } from 'calypso/components/loading-ellipsis';
 import StepWrapper from 'calypso/signup/step-wrapper';
-import {
-	fetchAutomatedTransferStatusOnce,
-	requestEligibility,
-} from 'calypso/state/automated-transfer/actions';
-import {
-	isFetchingAutomatedTransferStatus,
-	getEligibility,
-	EligibilityData,
-} from 'calypso/state/automated-transfer/selectors';
-import { getSiteDomain } from 'calypso/state/sites/selectors';
+import useWoopHandling from '../hooks/use-woop-handling';
```

Comment:
Could this name be a bit more specific? And/or the same as is defined in the hook?

`useWooCommerceOnPlansEligibility`?